crates/bin/node/src/initialization/grpc.rs

```diff
@@ -13,20 +13,21 @@ use sqlx::Postgres;
 use starknet_types::Unit;
 use tonic::{service::LayerExt, transport::Channel};
 
-use crate::{grpc_service::GrpcState, liquidity_sources::LiquiditySources};
+use crate::{app_state::AppState, liquidity_sources::LiquiditySources};
 
 use super::{Error, env_variables::EnvVariables};
 
+/// Create AppState that can be shared between gRPC and HTTP services
 #[instrument]
-pub async fn launch_tonic_server_task(
+pub async fn create_app_state(
```

Comment:
move this one in `initialization/app_state` or `initialization/mod.rs` so that it is not in the `grpc` file, as it is shared between both grpc and rest api